docs/understanding-radicle/glossary.md

```diff
@@ -21,162 +23,148 @@ changes committed to a [project][pr] history along with who made them and when.
-be published on Radicle. This is the only remote managed by Radicle that can be
-pushed to.
-
-`git push rad`
+## Radicle ID
+Unique shareable identifiers for peers and projects in the Radicle Network. 
 
 ## Radicle Link
-A peer-to-peer replication protocol built on Git. You can read more about it
-[here][rl].
+A peer-to-peer replication protocol built on Git. See [How it
+Works](how-it-works.md).
 
 ## Radicle network
-The network of peers that are replicating on the Radicle Link protocol.
+The network of peers that replicate and gossip data with Radicle Link.
 
 ## radicle-upstream
-A desktop application (graphic [user][us] interface, GUI) built to interact
-with and enable access to the Radicle Network and, initially, the primary
-end-[user][us] experience. However, it is one of many potential clients that
-[user][us]s can use to access the Radicle Network.
+A desktop application (graphic [user][us] interface, GUI) built to interact with
+and enable access to the Radicle Network and, initially, the primary
+end-[user][us] experience. However, it will be one of many potential clients
+that [user][us]s can use to access the Radicle Network.
 
 ## remote
-A remote copy of a [repository][re] with the same [project][pr] ID, tracked by
-the VCS. Remotes can be pulled/fetched from. Maintainers of a [repository][re]
-may choose to track the repositories of other owners. If the remote
-[repository][re] (or "remote," in Git terminology) is found to track other
-remotes, the tracking [repository][re] shall also transitively track those, up
-to a configurable N degrees out.
+Another peer's view of a project. Remotes can be pulled/fetched from.
+Maintainers of a project may choose to track other peer's views of their project
+by adding them as a remote. In the context of a project, maintainers of a
+repository may choose to track the views of other owners (this is called a
+remote in Git terminology: a named reference to a remote repository). If the
+remote repository is found to track other remotes, the tracking repository shall
+also transitively track those, up to a configurable N degrees out (currently in
+the works).
 
 ## repository
 A physical manifestation or working copy of a [project][pr], tied to a
 [user][us]'s machine. A [repository][re] in Radicle contains the owner's view
-and the view of any tracked peers. To publish a [repository][re] to the
-network, it must first be initialized as a [project][pr] or part of a
-[project][pr]. A [repository][re] can be addressed by a shareable URN, which is
+and the view of any tracked peers. To publish a [repository][re] to the network,
+it must first be initialized as a [project][pr] or part of a [project][pr]. A
+[repository][re] can be addressed by a shareable URN (See Radicle ID), which is
 then resolved to a physical location on the network, for the purpose of
-replication. This URN has the following form: `rad:git:<identity hash>`
+replication (See Radicle URNs)
```

Comment:
This definition is confused I feel. Needs work.

Reply:
Deleting and will focus on reworking the project definition